src/Question.js

```diff
@@ -10,19 +10,32 @@ class Question extends Component {
       option: PropTypes.string.isRequired,
       correct: PropTypes.bool.isRequired
     })),
-    onClick: PropTypes.func.isRequired
+    onClick: PropTypes.func.isRequired,
+    questionNumber: PropTypes.number.isRequired,
+    userHasAnswered: PropTypes.bool.isRequired
+  }
+  getQuestionClassName() {
+    const questionClassName = {
+      correct ? 'answered' : 'wrong-answer'
```

Comment:
This is the major issue, and one that will mean we have to re-work some things....

Where is correct coming from? The variable "correct" is not created or assigned anywhere in question so this will always come out as wrong-answer.

The key question here is... how do we check the answer? We may need to change what we're passing into <Question>. Currently we pass in a boolean as userHasAnswered, but we may need to remove those exclamation points, move them to line 38 in question, and change the prop userHasAnswered to userAnswer. This way we can check this.props.userAnswer in this function for whether it is correct or not.